ister_gui.py

```diff
@@ -1653,22 +1679,24 @@ def __init__(self, *args, **kwargs):
             self._exit(-1, 'Template file does not exist')
 
     def _init_actions(self):
-        telem_disclosure = TelemetryDisclosure()
-        startmenu = StartInstaller()
-        config_hostname = ConfigureHostname()
+        telem_disclosure = TelemetryDisclosure(1, 3)
+        startmenu = StartInstaller(2, 3)
+        config_hostname = ConfigureHostname(5, 9)
         confirm_disk_wipe = ConfirmDiskWipe()
         confirm_disk_wipe2 = ConfirmDiskWipe()
-        part_menu = PartitioningMenu()
-        automatic_device = SelectDeviceStep()
-        manual_part_device = SelectMultiDeviceStep()
-        manual_nopart_device = SelectDeviceStep()
+        part_menu = PartitioningMenu(3, 9)
+        automatic_device = SelectDeviceStep(3, 3)
+        manual_part_device = SelectMultiDeviceStep(4, 9)
+        manual_nopart_device = SelectDeviceStep(4, 9)
         terminal_cgdisk = TerminalStep()
-        set_mount_points = MountPointsStep()
-        bundle_selector = BundleSelectorStep()
-        confirm_user = ConfirmUserMenu()
-        user_configuration = UserConfigurationStep()
-        confirm_dhcp = ConfirmDHCPMenu()
-        static_ip_config = StaticIpStep()
+        set_mount_points = MountPointsStep(4, 9)
+        bundle_selector = BundleSelectorStep(6, 9)
+        confirm_user = ConfirmUserMenu(7, 9)
+        user_configuration = UserConfigurationStep(8, 10)
+        confirm_dhcp = ConfirmDHCPMenu(8, 9)
+        confirm_dhcp2 = ConfirmDHCPMenu(9, 10)
+        static_ip_config = StaticIpStep(9, 10)
+        static_ip_config2 = StaticIpStep(10, 11)
```

Comment:
9/9 and 11/11 is the `ConfirmStep()` which tells the user the setup is complete and asks the user to confirm if they want to begin the install. I assumed the user would be able to figure out it was the last step.